lang/en/facetoface.php

```diff
@@ -235,6 +235,7 @@
 $string['managerchanged'] = 'Your manager\'s email address has been changed.';
 $string['manageremail'] = 'Manager\'s email';
 $string['manageremailaddress'] = 'Manager\'s email address';
+$string['manageremaildisabled'] = 'Manager\'s email settings are not available if manager approvals are not enabled.';
```

Comment:
This message is only to explain why a settings section is empty. The settings api does not allow the setting to be merely disabled which would be a better experience. 

If you can suggest a positive message that still conveys the information, please provide it. I haven't been able to come up with one. 

If it's too confusing, I would just remove this message altogether and have the empty settings section.

Reply:
Some images of settings page if it helps create a better message.

![image](https://user-images.githubusercontent.com/36656753/215009809-bac5c6bf-6afb-42ff-b003-5e99725b1d1a.png)
![Screenshot from 2023-01-27 15-31-03](https://user-images.githubusercontent.com/36656753/215009839-27130bbc-1ca0-4d09-a323-59919c211163.png)

Reply:
No action taken, but happy for a suggestion on rewording this message.

Reply:
Thanks for the screenshots - both help paint the picture clearer. 

The top image looks good. I would prefer having a description under the heading, as opposed to an empty section.

The details under the "Manager Emails" section heading can be omitted. Instead, it'd be ideal if the description was added to the `enableapprovals` control, since that's the trigger point.

Reply:
Bit confused about what you are suggesting?

The settings API doesn't allow non editable elements such as a plain text element to be hidden conditionally. So the text explaining why manager's email settings are not there when approvals are disabled will always be shown to user.

> I would prefer having a description under the heading, as opposed to an empty section.

This is the current state. That's how Moodle renders it. The explanation text is the 'description' of that heading.

> The details under the "Manager Emails" section heading can be omitted. Instead, it'd be ideal if the description was added to the enableapprovals control, since that's the trigger point.

I don't want to append it to the description of the checkbox as I believe it would muddy the description of the functionality. 

Do you want me to move it as a text element under the enableapprovals checkbox element? This means the Manager's email section will just be empty with no explanation when the checkbox is ticked.

I thought it made sense to explain what is affecting the section, in the section providing information to the user of what is affecting it. For example if a bunch of settings were added between these elements, and someone checks they box. They may not notice the settings being hidden and then when they scroll down to the section, it will just be empty with no explanation why.

Reply:
> This is the current state. That's how Moodle renders it. The explanation text is the 'description' of that heading.

Yes, this response was targeting a previous response given where you mentioned "If it's too confusing, I would just remove this message altogether and have the empty settings section."

If this is the current state then I'm happy with it.

> I don't want to append it to the description of the checkbox as I believe it would muddy the description of the functionality.

That's a fair point, and as long as that control `enableapprovals` intuitively implies that the section `Manager Emails` is irrelevant, I'm happy for it to not be included. A loose comparison would be a switch, turning off power to a building. You wouldn't need to label "microwave will no longer work" next to the switch, because that's an implied effect from the control. 

Would it make more sense, and be more clear, to relabel that section "Manager approval emails"? (I'm not sure what other settings are under that group but if not then ignore this comment)

Reply:
I'm going to make the call that it is as clear as it can be without wasting more time on this. I'm happy as it is.